SmartPlaces.Facilities/lib/OntologyMapper/src/OntologyMappingManager.cs

```diff
@@ -102,5 +102,28 @@ public bool ValidateTargetOntologyMapping(IReadOnlyDictionary<Dtmi, DTEntityInfo
 
             return !invalidTargets.Any();
         }
+
+        public bool ValidateSourceOntologyMapping(IReadOnlyDictionary<Dtmi, DTEntityInfo> sourceObjectModel, out List<string> invalidSources)
```

Comment:
Could the naming of the `ValidateTargetOntologyMapping` method become `ValidateOntologyMapping` it seems the Target and Source methods only differ in variable names, but not actual behavior?

Reply:
No.. its just the object within the thing they're looking at.. hmm..